SpriteWander/entity/Entity.cs

```diff
@@ -263,11 +245,12 @@ protected void NextAnim()
             state = Texture.Normalise(state, out _);
             if (Texture.EndBehaviour(state) == AnimEvent.Reset)
             {
-                cycle = rng.Next(5, 20);
+                int maxCycle = state == Animation.Walk ? 30 : 20;
+                cycle = rng.Next(4, maxCycle);
```

Comment:
shouldn't we do something like `cycle = state == Animation.Walk ? 50 : rng.Next(4, maxCycle);` so that walking pokemon goes to the target position?

Reply:
i thought it would be fine if they didn't travel all the way every time because most of the time it would be a long travel so mixing it up was a bit nice

Reply:
well, something we can do is choose the next location closer to the pokemon with like a 2D normal distribution center around the pokemon (and reroll if the target is out of bounds). so we have shorter walking distance (can be controle with the normal deviation) and the pokemon shouldn't gather around the borders

Reply:
TRUE!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!